user_tools/src/spark_rapids_pytools/resources/templates/cluster_template/emr.ms

```diff
@@ -0,0 +1,30 @@
+{
+	"Cluster": {
+	  "Id": "j-123456789",
+	  "Name": "default-cluster-prop",
+	  "Status": {
+	    "State": "TERMINATED"
+	  },
+	  "Ec2InstanceAttributes": {
+	    "Ec2AvailabilityZone": "us-west-2a"
+	  },
+	  "InstanceGroups": [
```

Comment:
For EMR, it is possible to have different three groups "MASTER", "CORE", and "TASK".
The template here does not seem to handle if there is group with "TASK".
It is possible to handle this case by making an optional section in the moustache template and it would be turned on/off based on arguments.

If you find blocker issues handling this, there is a different route where we use JSON schema loader/validator.
You probably got some glimpses dealing with it when I implemented the new CLI. For example, a class like "`PropValidatorSchemaCamel`"  is something similar. 
We can discuss offline.

Reply:
I tested the cost calculation having 3 core nodes vs 2 core node + 1 task node. The pricing is same because we map both `TASK` and `CORE` to be `WORKERS`. 

Additionally, from the event logs, I do not think there is a way to distinguish between them. Hence in core tools also, we will get 3 executor nodes  in both cases.